vars/pluginUtils.groovy

```diff
@@ -0,0 +1,88 @@
+def runPluginTests(plugins, envName, markerName, slackChannel, slackUrl) {
+    def lockName = "${env.JOB_NAME}-${envName}"
+    def pluginNames = plugins.keySet()
+
+    if (markerName != "") {
+        lockName = lockName + "-${markerName}"
+    }
+
+    lock(lockName) {
+        timeout(time: 150, unit: "MINUTES") {
```

Comment:
Can we make timeout time a variable. 150 minutes by default is OK for default though